### DISCUSSION TITLE
Digit fomat with HEX

### DISCUSSION THREAD

**Discussion by KKimj** [AntlerVC/firetable#735] [Support / Q&A] [answered]
I want to See number with HEX, hexadecimal format.

<img width="345" alt="image" src="https://user-images.githubusercontent.com/40026920/182544062-2f8e5318-fab4-4bfa-bc67-d1cbc0385ce8.png">

e.g., 0xFF001122

I want to Read / Write with hexadecimal number! 
<img width="631" alt="image" src="https://user-images.githubusercontent.com/40026920/182544190-26782f73-aa4f-49f6-a960-e3c39c98316f.png">

Could I Do this with Rowy?
Please let me know
Thanks!!

**Comment by harinij** [comment, marked as answer]
The number field currently doesn't support this. You can use a derivative column instead to read the data from the number field and display it in a HEX format. Writing can also be achieved by either using an extension. But yeah these are not directly at field level and require cloud functions. 

I have converted this discussion into an issue to support such a type in the future.

**Reply by KKimj** [reply]
Thanks Thanks!! 
Appreciate for your support!
